### PR TITLE
Inherited from plotter instead of wrapping it

### PR DESCRIPTION
**Pull request by nabobalis**
This is a major breaking change.

### CODE REVIEW

File: sunkit_pyvista/plotter.py [hunk 1]
```diff
@@ -45,16 +45,16 @@ class SunpyPlotter:
...
         if coordinate_frame is not None and obstime is not None:
             msg = "Only coordinate_frame or obstime can be specified, not both."
             raise ValueError(msg)
         if coordinate_frame is None:
             coordinate_frame = HeliocentricInertial(obstime=obstime)
         self._coordinate_frame = coordinate_frame
-        self._plotter = pv.Plotter(**kwargs)
-        self.camera = self._plotter.camera
         self.all_meshes = {}
+        self._initialized = True
```
Comment (Cadair) [inline sunkit_pyvista/plotter.py:57]: What's this do?

Comment (nabobalis) [reply sunkit_pyvista/plotter.py:57]: Err, not a clue but its the base plotter was setup upstream and I seemed to need it.

Comment (nabobalis) [reply sunkit_pyvista/plotter.py]: Turns out I am wrong. This was not needed. I'm just stupid.

**Review by Cadair — APPROVED** [review]
If I works then good to me.

**Pull request merged**